src/WorkflowsRules/Cli/cli.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+use function Baka\appPath;
+use Dotenv\Dotenv;
+use Phalcon\Cli\Console as ConsoleApp;
+use Phalcon\Di\FactoryDefault\Cli as CliDI;
+use Phalcon\Loader;
+use Swoole\Runtime;
+
+require '/app/vendor/autoload.php';
```

Comment:
@FredPeal this path cant be hardcoded , need to be based on path

Reply:
yes, but this cli is for test local use, not production, for production you must use the cli by canvas_core